lasagne/layers/pool.py

```diff
@@ -18,18 +17,18 @@
 class MaxPool2DLayer(Layer):
     def __init__(self, incoming, ds, ignore_border=False, **kwargs):
         super(MaxPool2DLayer, self).__init__(incoming, **kwargs)
-        self.ds = ds # a tuple
+        self.ds = ds  # a tuple
         self.ignore_border = ignore_border
 
     def get_output_shape_for(self, input_shape):
-        output_shape = list(input_shape) # copy / convert to mutable list
+        output_shape = list(input_shape)  # copy / convert to mutable list
 
         if self.ignore_border:
-            output_shape[2] = int(np.floor(float(output_shape[2]) / self.ds[0]))
-            output_shape[3] = int(np.floor(float(output_shape[3]) / self.ds[1]))
+            output_shape[2] = int(np.floor(float(output_shape[2])/self.ds[0]))
+            output_shape[3] = int(np.floor(float(output_shape[3])/self.ds[1]))
         else:
-            output_shape[2] = int(np.ceil(float(output_shape[2]) / self.ds[0]))
-            output_shape[3] = int(np.ceil(float(output_shape[3]) / self.ds[1]))
+            output_shape[2] = int(np.ceil(float(output_shape[2])/self.ds[0]))
+            output_shape[3] = int(np.ceil(float(output_shape[3])/self.ds[1]))
```

Comment:
Yes, removing that whitespace isn't PEP8.  The PEP8 tool has [E226](http://pep8.readthedocs.org/en/latest/intro.html#error-codes), which requires whitespace around arithmetic operators, but it's turned off by default.  There's certainly no rule that says that this whitespace should be removed, so this change arguably isn't in the scope of the PR.